### PR TITLE
Board Membership and lead developer SEP

### PR DESCRIPTION
**Pull request by ehsteve**
The purpose of this SEP is to create a record of the board membership as well as to name the lead developer. This information will also be posted on the sunpy website but this will be the foundational document.

### CODE REVIEW

File: SEP-0006.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# Board Membership
+The SunPy board is composed of the following members (in alphabetical order)
+
+| Name                          | Role              | Start       |  Expiration |
+|-------------------------------|-------------------|-------------|-------------|
+| Steven Christe                | Chair             | 17 Mar 2014 | 31 Dec 2016 |
+| Russell Hewett                |                   | 17 Mar 2014 | 31 Dec 2015 |
+| Andrew Inglis                 | Secretary         | 17 Mar 2014 | 31 Dec 2015 |
+| Jack Ireland                  |                   | 17 Mar 2014 | 31 Dec 2016 |
+| Stuart Mumford                |                   | 17 Mar 2014 | 31 Dec 2015 |
```
Comment (Cadair) [inline SEP-0006.md]: Role: Executive Director?

Comment (ehsteve) [inline SEP-0006.md]: That is not a board role I think. The board appoints a director/lead developer but that person does not need to be on the board.

Comment (Cadair) [inline SEP-0006.md]: true.

Maybe make the lead developer section say ED / lead ?

Comment (ehsteve) [inline SEP-0006.md]: Good point. SEP-0002 only references the executive director. Fixed.

File: SEP-0006.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+# Abstract
+The purpose of this document is contains the membership information of
+the SunPy board, including names as well as membership start, and expiration
+date and board roles. Additionally this document will name the current Executive
+Director.
+
+# Detailed Description
+There is no official place to list the current membership of the SunPy board
+or the lead developer. This document shall be referenced to by the sunpy board
+founding document (SEP-00002) and this information shall also be posted on the
```
Comment (Cadair) [inline SEP-0006.md]: hyperlink to SEP2?

**Comment by ehsteve** [comment]
Accepted by a meeting of the board on 7-Dec-2015 (see https://github.com/sunpy/sunpy/wiki/Minutes-of-SunPy-Board-Meeting-12-07-15).

**Pull request merged**